src/cljs/vanilla/widgets/scatter_chart.cljs

```diff
@@ -0,0 +1,55 @@
+(ns vanilla.widgets.scatter-chart
+  (:require [reagent.core :as r]
+            [reagent.ratom :refer-macros [reaction]]
+            [vanilla.widgets.make-chart :as mc]))
+
+(defn plot-options
+  [chart-config data options]
+
+  ;(.log js/console (str "column/plot-options " chart-config))
+
+  {:xAxis       {:title {:text          (get-in data [:data :src/x-title] "x-axis")
+                         :allowDecimals (get options :viz/x-allowDecimals false)}
+                 :startOnTick true
+                 :endOnTick true
+                 :showLastLabel true}
+
+
+   :yAxis       {:title {:text          (get-in data [:data :src/y-title ] "y-axis")
+                         :allowDecimals (get options :viz/y-allowDecimals false)}}
+
+   :legend      {:align         "left"
```

Comment:
the legend seems to draw in the middle of the graph. let's play with these parameters get it somewhere the graph and tooltips don't cover